doc/metals.txt

```diff
@@ -388,39 +388,39 @@ handle it.
 COMMANDS                                                       *metals-commands*
 
 The following commands are provided by nvim-metals:
-  * |MetalsAmmoniteStart|
-  * |MetalsAmmoniteStop|
+  * |MetalsStartAmmonite|
```

Comment:
Sort of a nitpick, but if we are going to change this all around, can we keep them alphabetical and rearrange them?

Reply:
Done